django_irods/storage.py

```diff
@@ -434,7 +434,7 @@ def checksum(self, full_name, force_compute=True):
         return checksum
 
     def url(self, name, url_download=False, zipped=False, aggregation=False):
-        reverse_url = reverse('django_irods_download', kwargs={'path': name})
+        reverse_url = reverse('rest_download', kwargs={'path': name})
```

Comment:
@sblack-usu This url is used for internal download. There is a separate rest download URL named ```rest_download```. Are you sure this change is correct and needed?

Reply:
Thanks for pointing this out.  I changed it to rest download so we could use credentials outside of the CSRF token.  Perhaps we should look at alternative approaches.  Basically, we need to be able to download files with the urls in the resourcemap using rest authentication methods.

Reply:
I am not sure precisely what this will do, given current performance tuning that is based upon the other alternative. And it's subject to change as I tune up upload/download. Probably harmless but I am unsure.

Reply:
I've been using it with the hs_rdf client I'm writing and have tested this thoroughly on dev-hs-1, the gui behaves the same.  The download implementation should be the same, this just allows credentials outside of CSRF.  Please let me know if you find this conflicts with the upload/download tuning, I'm confident it shouldn't but do not know for certain.